File: Atividade III/problema2.py
x = float(input("Nota 1: "))
y = float(input("Nota 2: "))
z = float(input("Nota 3: "))

media = (lambda x, y, z: (x + y + z) / 3)
aprovado = (lambda x: "Reprovado" if x < 6 else "Aprovado")

print(aprovado(media(x, y, z)))

File: Atividade IX/problema2.hs
media :: Float -> Float -> Float -> Bool
media = (\x y z -> ((x + y + z) / 3) >= 6)

main = do
  putStrLn "Digite a nota:"
  a <- getLine
  putStrLn "Digite a nota:"
  b <- getLine
  putStrLn "Digite a nota:"
  c <- getLine
  let x = (read a :: Float)
  let y = (read b :: Float)
  let z = (read c :: Float)
  print (media x y z)